overlord/state/state.go

```diff
@@ -139,6 +142,7 @@ func (s *State) Modified() bool {
 
 // Lock acquires the state lock.
 func (s *State) Lock() {
+	s.lockStart = osutil.GetLockStart()
```

Comment:
shouldn't this be done with the lock taken? otherwise just the tentative themselves to lock will race/dirty lockStart